wolfssl/wolfcrypt/rsa.h

```diff
@@ -353,6 +353,7 @@ WOLFSSL_API int wc_RsaExportKey(RsaKey* key,
                                 byte* q, word32* qSz);
 
 WOLFSSL_API int wc_RsaKeyToPublicDer(RsaKey*, byte* output, word32 inLen);
+WOLFSSL_API int wc_RsaKeyToPublicDer_ex(RsaKey* key, byte* output, word32 inLen, int with_header);
```

Comment:
For FIPS the rsa.h header is different (the certified version) so this API def won't exist. I recommend placing this new API def into asn_public.h instead. Or you will need to wrap the new asn.c in `#if !defined(HAVE_FIPS) && !defined(HAVE_SELFTEST)`.